content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -0,0 +1,516 @@
+
+## Prerequisites
+
+- A Unity **Plus** or **Pro** license. Your license is used to activate Unity on the Codemagic build server so the iOS and Android projects can be exported. The license is then returned before the workflow builds and publishes your apps to the App Store and Google Play.
+- If you are publishing to the Apple App Store you will need an active membership for the Apple Developer Program.
+- For publishing to Google Play you will require an active Google Play Developer account.
+
+{{<notebox>}}
+**Important!** The Unity SDK is not included in our standard base images. Please contact us [here](https://codemagic.io/contact/) to request access to the Unity SDK base image.
+{{</notebox>}}
+
+## Adding your Unity project to Codemagic
+
+It is possible to add repositories from Github, Gitlab, Bitbucket or any Git based repository. Please refer to the following documentation about adding apps from these sources.
+
+- [Adding apps from GitHub](../getting-started/github/).
+- [Adding apps from GitLab](../getting-started/gitlab/).
+- [Adding apps from Bitbucket](../getting-started/bitbucket/).
+- [Adding apps from other repositories](../getting-started/other/).
+
+## Setting up your workflow
+
+These are the key steps in setting up your workflow for building Unity mobile apps: 
+
+1. On the Applications page in Codemagic click the 'Add application' button and follow the on-screen instructions to add your Unity project.
+2. Add environment variables for Unity as described [here]({{< ref "#unity-variables" >}}).
+3. Add environment variables for iOS code signing as described [here]({{< ref "#ios-variables" >}}).
+4. Add environment variables for Android code signing as described [here]({{< ref "#android-variables" >}}).
+5. Add a build script to Unity as explained [here]({{< ref "#unity-build-script" >}}).
+6. Add a post-processing script to Unity for iOS builds as described [here]({{< ref "#post-processing-script" >}}).
+7. Set the iOS bundle identifier in Unity as described [here]({{< ref "#bundle-identifier" >}}).
+8. Configure Custom Gradle templates as described [here]({{< ref "#custom-gradle-templates" >}}).
+9. Add a script for license activation, mobile app export and license return [here]({{< ref "#license-activation" >}}).
+10. Create a codemagic.yaml workflow configuration using the Unity template and configure with your details as shown [here]({{< ref "#workflow-configuration" >}}).
+
+## Environment variables for Unity {#unity-variables}
+
+You will need to set the following environment variables for Unity specific values in Codemagic: 
+
+`UNITY_HOME`, `UNITY_SERIAL`, `UNITY_USERNAME` and `UNITY_PASSWORD`.
+
+If you are using a Team in Codemagic, you can add these as global environment variables for your team by clicking on Teams > your Team name and then clicking on Global variables and secrets. Otherwise you can add the environment variables at application level by clicking the Environment variables tab.
```

Comment:
```suggestion
If you are using a Team in Codemagic, you can add these as global environment variables for your team by clicking on **Teams > your Team name** and then clicking on **Global variables and secrets**. Otherwise, you can add the environment variables at the application level by clicking the **Environment variables** tab.
```